tests/testthat/test-utils.R

```diff
@@ -153,9 +153,19 @@ test_that("h_get_cov_default works correctly", {
 # h_confirm_large_levels ----
 
 test_that("h_confirm_large_levels errors for large number", {
+  skip_if(interactive())
```

Comment:
why do we need this? test should still be ok to run interactively I think

Reply:
(it will ask you yes/no and if you provided wrong answer tests might fail. but we can make `askpass` always return FALSE use mockery

Reply:
ah ok then good as is, most important is that non-interactive tests run